devices/tuya/_TZE204_ac0fhfiq_energy_meter.json

```diff
@@ -0,0 +1,198 @@
+          "name": "attr/name"
+        },
+        {
+          "name":"attr/swversion",
+          "read":{
+            "at":"0x0001",
+            "cl":"0x0000",
+            "ep":1,
+            "fn":"zcl"
+          },
+          "parse":{
+            "at":"0x0001",
+            "cl":"0x0000",
+            "ep":1,
+            "fn":"zcl",
+            "script":"tuya_swversion.js"
+          }
+        },
+        {
+          "name": "attr/type"
+        },
+        {
+          "name": "attr/uniqueid"
+        },
+        {
+          "name": "config/on"
+        },
+        {
+          "name": "config/reachable"
+        },
+        {
+          "name": "state/consumption",
+          "read": {
+            "fn": "tuya"
+          },
+          "parse": {
+            "dpid": 1,
+            "eval": "Item.val = Attr.val * 10;",
+            "fn": "tuya"
+          },
+          "default": 0
+        },
+        {
+          "name": "state/consumption_2",
+          "read": {
+            "fn": "none"
+          },
+          "parse": {
+            "dpid": 2,
+            "eval": "Item.val = Attr.val * 10;",
+            "fn": "tuya"
+          },
+          "default": 0
+        },
+        {
+          "name": "state/lastupdated"
+        }
+      ]
+    },
+    {
+      "type": "$TYPE_POWER_SENSOR",
+      "restapi": "/sensors",
+      "uuid": [
+        "$address.ext",
+        "0x01",
+        "0x0b04"
+      ],
+      "items": [
+        {
+          "name": "attr/id"
+        },
+        {
+          "name": "attr/lastannounced"
+        },
+        {
+          "name": "attr/lastseen"
+        },
+        {
+          "name": "attr/manufacturername"
+        },
+        {
+          "name": "attr/modelid"
+        },
+        {
+          "name": "attr/name"
+        },
+        {
+          "name":"attr/swversion",
+          "read":{
+            "at":"0x0001",
+            "cl":"0x0000",
+            "ep":1,
+            "fn":"zcl"
+          },
+          "parse":{
+            "at":"0x0001",
+            "cl":"0x0000",
+            "ep":1,
+            "fn":"zcl",
+            "script":"tuya_swversion.js"
+          }
+        },
+        {
+          "name": "attr/type"
+        },
+        {
+          "name": "attr/uniqueid"
+        },
+        {
+          "name": "config/on"
+        },
+        {
+          "name": "config/reachable"
+        },
+        {
+          "name": "state/current",
+          "read": {
+            "fn": "none"
+          },
+          "parse": {
+            "dpid": 6,
+            "eval": "Item.val = ((ZclFrame.at(8) << 16) | (ZclFrame.at(9) << 8) | ZclFrame.at(10)) / 1000;",
+            "fn": "tuya"
+          },
+          "default": 0
+        },
+        {
+          "name": "state/direction",
+          "parse": {
+            "dpid": 102,
+            "eval": "if (Attr.val == 0) { Item.val = 'consuming' } else { Item.val = 'producing' };",
+            "fn": "tuya"
+          }
```

Comment:
We had a look at the PR in the last meeting and came to the conclusion that the idea of `state/direction` as string is difficult. Therefore the proposal is to not include this item, since the information of the direction is already exposed via `state/power` which is a signed integer, and positive/negative values express the direction.

Reply:
Removed